src/diffpy/utils/transforms.py

```diff
@@ -4,12 +4,10 @@
 import numpy as np
 
 wavelength_warning_emsg = (
-    "INFO: no wavelength has been specified. You can continue "
-    "to use the DiffractionObject but some of its powerful features "
-    "will not be available. To specify a wavelength, set "
-    "diffraction_object.wavelength = [number], "
-    "where diffraction_object is the variable name of you Diffraction Object, "
-    "and number is the wavelength in angstroms."
+    "No wavelength has been specified. You can continue to use the DiffractionObject, but "
+    "some of its powerful features will not be available. "
+    "To specify a wavelength, if you have do = DiffractionObject(xarray, yarray, 'tth'), "
+    "you may set do.wavelength = 1.54 for a wavelength of 1.54 angstroms."
```

Comment:
`you may set do.wavelength = 1.54 for a wavelength of 1.54 angstroms` is clearer than what I had earlier.